bigquery/samples/client_query_destination_table_cmek.py

```diff
@@ -24,14 +24,14 @@ def client_query_destination_table_cmek(client, table_id):
     # TODO(developer): Set table_id to the ID of the destination table.
     # table_id = "your-project.your_dataset.your_table_name"
 
-    job_config = bigquery.QueryJobConfig()
-    job_config.destination = table_id
-
     # Set the encryption key to use for the destination.
     # TODO(developer): Replace this key with a key you have created in KMS.
-    kms_key_name = "projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}".format(
-        "cloud-samples-tests", "us", "test", "test"
-    )
+    # kms_key_name = "projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}".format(
+    #     your-project, location, your-ring, your-key
+    # )
```

Comment:
`kms_key_name` fixture was created and used across `*_cmek` files. Nevertheless, there is a question about `TODO(developer):` commented lines. Should it represent the whole line at ones, or it can use `.format()`?

Reply:
Since it's such a long string, I think `.format()` is more readable. I'm fine either way.